src/admin.c

```diff
@@ -234,8 +234,13 @@ static bool admin_set(PgSocket *admin, const char *key, const char *val)
 	if (admin->admin_user) {
 		ok = set_config_param(key, val);
 		if (ok) {
+			PktBuf *buf = pktbuf_dynamic(256);
+			if (strstr(key, "_tls_") != NULL) {
```

Comment:
This is a bit fishy. ;-) I haven't thought about a better idea though.

Reply:
I agree, I could also remove this check and call `sbuf_tls_setup()` always.